darwin/pytorch/utils.py

```diff
@@ -0,0 +1,232 @@
+import os
+import errno
+import torch
+import numpy as np
+import matplotlib.pyplot as plt
+import json
+from pycocotools import mask as coco_mask
+from pathlib import Path
+from typing import TYPE_CHECKING, Any, Dict, List, Optional
```

Comment:
you don't seem to be using `TYPE_CHECKING` anywhere.

Reply:
True. Same for `Any`, `Dict`, `List`. Removed those four.